Tests/RelativityTests/ViewPositionTests.swift

```diff
@@ -62,55 +35,97 @@ class ViewPositionTests: XCTestCase {
+        runTestOnAllScreenScales {
+            setUpWithCurrentScreenScale()
+            XCTAssertEqualWithAccuracy(subview1.topLeft |--| subview2.bottomRight, subview2.bottomRight |--| subview1.topLeft, accuracy: PixelRounder.significantPrecision)
+        }
     }
     
     public func test_measureDistance_assertsOriginIsOnPixelBoundary() {
-        subview1.frame.origin.x += 0.54321
-        subview2.frame.origin.y += 0.12345
-        
-        let viewOriginRoundedToPixelAssertMessage = "Measuring distance with a ViewPosition whose origin is not pixel aligned!"
-        var assertBodyCalledForOriginPixelAlignmentTest = false
-        Relativity.ErrorHandler.customAssertBody = { condition, message, _, _ in
-            if message == viewOriginRoundedToPixelAssertMessage {
-                XCTAssertFalse(condition)
-                assertBodyCalledForOriginPixelAlignmentTest = true
+        runTestOnAllScreenScales {
+            setUpWithCurrentScreenScale()
+            subview1.frame.origin.x += 0.54321
+            subview2.frame.origin.y += 0.12345
+
+            let viewOriginRoundedToPixelAssertMessage = "Measuring distance with a ViewPosition whose origin is not pixel aligned!"
+            var assertBodyCalledForOriginPixelAlignmentTest = false
+            Relativity.ErrorHandler.customAssertBody = { condition, message, _, _ in
+                if message == viewOriginRoundedToPixelAssertMessage {
+                    XCTAssertFalse(condition)
+                    assertBodyCalledForOriginPixelAlignmentTest = true
+                }
             }
+
+            let _ = subview1.left |--| subview2.right
+            XCTAssertTrue(assertBodyCalledForOriginPixelAlignmentTest)
         }
-        
-        let _ = subview1.left |--| subview2.right
-        XCTAssertTrue(assertBodyCalledForOriginPixelAlignmentTest)
     }
     
     public func test_measureDistance_ignoresTransforms() {
-        let preTransformDistance = subview1.bottom |--| subview2.top
-        
-        subview1.transform = CGAffineTransform(translationX: 50.0, y: -11.0)
-        subview2.transform = CGAffineTransform(scaleX: 2.0, y: -1.2)
-        
-        XCTAssertEqual(subview1.bottom |--| subview2.top, preTransformDistance)
+        runTestOnAllScreenScales {
+            setUpWithCurrentScreenScale()
+            let preTransformDistance = subview1.bottom |--| subview2.top
+
+            subview1.transform = CGAffineTransform(translationX: 50.0, y: -11.0)
+            subview2.transform = CGAffineTransform(scaleX: 2.0, y: -1.2)
+
+            XCTAssertEqual(subview1.bottom |--| subview2.top, preTransformDistance)
+        }
     }
     
     // MARK: Private Properties
     
     private let window = UIWindow()
     private let view = UIView()
-    private let subview1 = UIView()
-    private let subview2 = UIView()
+    private var subview1 = UIView()
```

Comment:
to avoid polluting the `bounds` of these views between different screen scale runs we now re-create these views